src/standard_library.md

```diff
@@ -1,6 +1,3 @@
 ## Standard Library
 
-The `alan` Standard Library is still a work in progress and barely complete enough to make non-trivial programs with, at the moment. The intention is to include many libraries such that "normal" usage of the language only requires libraries for access to third-party services, and then only as a convenience.
-
-**Modules without an asterisk next to their name already exist. Modules with one asterisk should be built before any reachout to potential customers. Modules with two asterisks should be built before the OSS release. Modules with three asterisks should be built before the SaaS launch.**
-
+The `alan` Standard Library is still small and barely complete enough to make non-trivial programs with, at the moment. We intend to follow the "batteries included" ethos of Python and Java and include many libraries such that "normal" usage of the language only requires libraries for access to third-party services, and then only as a convenience.
```

Comment:
let's add, if you are interested in contributing please drop as a line on discord?

Reply:
:+1: